Jenkinsfile

```diff
@@ -113,7 +115,7 @@ pipeline {
         GIT_AUTHOR_EMAIL = 'mc.stanislaw@gmail.com'
         GIT_COMMITTER_NAME = 'Stan Jenkins'
         GIT_COMMITTER_EMAIL = 'mc.stanislaw@gmail.com'
-        MULTIARCH_DOCKER_TAG = 'multiarch-ocaml-4.14-v2'
+        MULTIARCH_DOCKER_TAG = 'multiarch-ocaml-4.14-v2-and-cmdliner'
```

Comment:
I don't know anything about Jenkins, but what's the reason for the tag change?

Reply:
I needed to re-build the multiarch images to add cmdliner. Because they don't install the developer-only dependencies like `ocamlformat`, I guess they didn't have it already.

This was a headache, and it's definitely non-obvious how the multiarch build works, see #1480